frontend/Dockerfile

```diff
@@ -18,12 +18,21 @@ RUN pnpm install --ignore-scripts
 COPY . /work
 RUN pnpm run build -- --configuration=${CONFIGURATION}
 
-FROM nginxinc/nginx-unprivileged:latest
+FROM caddy:2.9-alpine
 
 EXPOSE 8080
 
-COPY nginx.conf /etc/nginx/nginx.conf
-COPY --from=build /work/dist/capellacollab /usr/share/nginx/html
-USER root
-RUN chmod -R 777 /var/log/nginx
-USER nginx
+RUN addgroup -S appgroup && adduser -S appuser -G appgroup \
```

Comment:
What do we need the `appuser` / `appgroup` for? The container doesn't run as the `appuser` anyway (we run it with a random user id)

Reply:
I agree we don't need the group, but didn't the previous nginx image do the exact same thing?

Reply:
As far as I can see, the previous nginx image did just as nginx user, which was probably used to indicate a non-root context. We also don't need the user itself.